src/backend/mi2/mi2.ts

```diff
@@ -576,6 +619,26 @@ export class MI2 extends EventEmitter implements IBackend {
 		return this.sendCommand("break-condition " + bkptNum + " " + condition);
 	}
 
+	setLogPoint(bkptNum, command): Thenable<any> {
+		const regex = /{([a-z0-9A-Z-_\.\>\&\*\[\]]*)}/gm;
+		let m;
+		let commands = "";
+
+		while ((m = regex.exec(command))) {
+			if (m.index === regex.lastIndex) {
+				regex.lastIndex++;
+			}
+
+			m.forEach((match, groupIndex) => {
+				if(groupIndex === 1){
+					commands += "\"print " + match + "\" ";
```

Comment:
probably want to just access `m[1]` instead of iterating and checking for an exact index?

might want to escape `match` as command argument as well?